types/koa-multer/README.md

```diff
@@ -0,0 +1,22 @@
+## koa-multer type defintions
```

Comment:
A custom `README.md` won't be published -- you could put a comment in `index.d.ts` instead.